databricks/koalas/indexes.py

```diff
@@ -588,6 +588,48 @@ def dropna(self):
         kdf = DataFrame(internal)
         return Index(kdf) if type(self) == Index else MultiIndex(kdf)
 
+    def nunique(self, dropna=True):
```

Comment:
@RainFung sorry for my late response. There's `Series.nunique` implementation. Can we move that implementation to `IndexOpsMixin` to support this in Index?

Reply:
OK, i will implement it based on IndexOpsMixin